### DISCUSSION TITLE
2023-08-02

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#229] [daily_meeting]
### 오늘의 할 일
- 공통
  - API 변경사항 개발

### 내일의 할 일
- 발표 준비(히이로, 노아)

- 프론트엔드
  - 웹접근성, 타이머(룩소)
  - UX 개선(엽토)

- 백엔드
  - 폴링, SSE, 소켓에 대해 논의해보기
  - 개발 서버 구축 시도

### 언젠가 할 일
- 소켓, 폴링 양방향 통신 어떻게 할지
- 기획 변경된 이유 정리
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

**Comment by jaehee329** [comment]
### 체크인
7점 - 무난하면 7점입니다. 오전 시간 활용은 못하겠네요 오늘은 ㅠ

### 체크아웃
9점 - 밥도 잘 먹고 코딩도 좀 해서 기분 좋습니다

**Comment by nlom0218** [comment]
### 체크인

10점 - 하나도 안 아픕니다. 피곤하지 않습니다.

### 체크아웃

9점 - 점심에 먹다남은 피자가 생각나요

**Comment by aak2075** [comment]
### 체크인

10점 - 잘자서 컨디션도 좋고 애슐리 뿌수러갑시다💪

### 체크아웃

9점 - 점심에 좀 더 먹을걸...

**Comment by woosung1223** [comment]
### 체크인
8점 - very sleepy

### 체크아웃
8점 - very sleepy 하지만 할 일은 어느정도 해서 좋네요

**Comment by woo-jk** [comment]
### 체크인
7점 - 컨디션이 완전히 돌아오진 않았따

### 체크아웃
8점 - 호화스러운 점심

**Comment by yeopto** [comment]
### 체크인

10점 - 어제 농구를하고 땀을 사우나 간 것처럼 빼서 뭔가 개운하고, 고급스러운 스벅 아메리카노 맛있네요.

### 체크아웃

8점 - 할게 많아졌어요 ㅠㅠ

**Reply by aak2075** [reply]
갓생ㄷㄷㄷ

**Reply by woosung1223** [reply]
저녁에는 농구하고 낮에는 개발하고 아침에는 스벅마시는남자

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 평범한데 애슐리 가서 기분이 좋아져서 8점입니다!

### 체크아웃
7점 - mockMvc 테스트 빨리 숙지하겠습니다...! 낼은 발표준비해야되는데 살짝 걱정되네요
